src/components/FormContainer/index.js

```diff
@@ -0,0 +1,77 @@
+import Form from "../Form";
+import List from "../List";
+import "./Container.css";
+import React, { useState, useEffect } from "react";
+
+function FormContainer() {
+  const [list, setList] = useState([]);
+
+  function Task(title = "", completed = false) {
+    this.id = Math.random().toString(36).substr(2, 18);
+    this.title = title;
+    this.completed = completed;
+  }
+
+  const InitialItems = [
+    new Task("TaskInicial1"),
+    new Task("TaskInicial2"),
+    //new Task("tarea 3"),
+    //new Task("tarea 4"),
+    //new Task("tarea 5"),
+  ];
+
+  useEffect(() => {
+    setList(InitialItems);
+    console.log(InitialItems);
+  }, []);
+
+  /*useEffect(() => {
+    const miStorage = window.localStorage;
+  }, [list]);*/
+
+  const _deleteTask = (itemToDelete) => {
```

Comment:
avoid function names like this adding a '_' symbol before the word

you want to reserve the use of that for render function handlers - functions that show certain JSX